fe/fe-core/src/main/java/org/apache/doris/analysis/OutFileClause.java

```diff
@@ -156,6 +158,7 @@ public class OutFileClause {
     private TFileFormatType fileFormatType;
     private long maxFileSizeBytes = DEFAULT_MAX_FILE_SIZE_BYTES;
     private boolean deleteExistingFiles = false;
+    private String fileSuffix = null;
```

Comment:
```suggestion
    private String fileSuffix = "";
```
I think there should not be null. Otherwise, `sinkOptions.setFileSuffix(fileSuffix);` it will give a `null` value to sinkOptions.

Reply:
thanks, modified